src/popup/router/pages/AccountPassword.vue

```diff
@@ -34,6 +34,8 @@ import { MINPASSWORDLENGTH } from '../../utils/constants';
 import Password from 'vue-password-strength-meter';
 import { mapGetters } from 'vuex'
 import { redirectAfterLogin } from '../../utils/helper';
+const Cryptr = require('cryptr');
+const cryptr = new Cryptr('myTotalySecretKey');
```

Comment:
@vmangelovv 

Is this key needed for decryption?
- If not we can use random buffer for seed here.
- Else why don't we use the user's password here?

Reply:
This is fixed in https://github.com/vmangelovv/aepp-waellet/commit/972a6c713f8da8a665eac47952e8c6f660ca7768. Now we are using webcrypto and user's password for encryption and decryption of the seed